edx_repo_tools/repo_checks/repo_checks.py

```diff
@@ -157,6 +157,72 @@ def dry_run(self):
         raise NotImplementedError
 
 
+class EnsureNoAdminOrMaintainTeams(Check):
+    """
+    Teams should not be granted `admin` or `maintain` access to a repository unless the access
+    is exceptions and it is noted here.  All other `admin` and `maintain` access is downgraded to
```

Comment:
```suggestion
    is an exception and it is noted here.  All other `admin` and `maintain` access is downgraded to
```

Not sure if  you meant "in exceptions" or just missed an "an".